packages/web/src/scss/components/Modal/index.html

```diff
@@ -761,4 +761,172 @@ <h2 id="example_disabled_backdrop_click_title" class="ModalHeader__title">Modal
 
 </section>
 
+<section class="docs-Section spirit-feature-modal-enable-uniform-dialog">
+  <h2 class="docs-Heading">Feature Flag: Uniform Modal on Mobile</h2>
+
+  <div class="docs-Stack docs-Stack--start">
+
+    <button
+      type="button"
+      class="Button Button--primary Button--medium"
+      data-spirit-toggle="modal"
+      data-spirit-target="#example_uniform"
+      aria-controls="example_uniform"
+      aria-expanded="false"
+    >
+      Open Modal
+    </button>
+
+    <button
+      type="button"
+      class="Button Button--primary Button--medium"
+      data-spirit-toggle="modal"
+      data-spirit-target="#example_docked"
+      aria-controls="example_docked"
+      aria-expanded="false"
+    >
+      Open Docked Modal (mobile only)
+    </button>
+
+    <!-- Modal: start -->
+    <dialog id="example_uniform" class="Modal" aria-labelledby="example_uniform_title">
```

Comment:
Should't be IDs in kebab-case as we agreed on dlouha temata?

Reply:
Indeed, they should. I would rather change them all once all Modal updates are done—a lot of rebasing can be expected down the road and the fewer changes are all over the place, the better.

Reply:
Eventually, I changed it right away, rebase is still quite safe.